docs/configuration.md

```diff
@@ -694,13 +694,13 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.executor.logs.rolling.strategy</code></td>
-  <td>(none)</td>
+  <td>""(disabled)</td>
   <td>
     Set the strategy of rolling of executor logs. By default it is disabled. It can
-    be set to "time" (time-based rolling) or "size" (size-based rolling). For "time",
+    be set to "time" (time-based rolling) or "size" (size-based rolling) or ""(empty string). For "time",
```

Comment:
Please put space after quotes here and above. (empty string) isn't helpful, (disabled) is. Then you can remove the sentence at the end that you added.

Reply:
@srowen Thank you for your advice.
I had follow your advice and adjust